tests/test_models.py

```diff
@@ -11,25 +11,25 @@ def test_list_backbone_names():
     assert len(list_backbone_names()) > 0
 
 
-def test_get_backbone():
-    for name in list_backbone_names():
-        with tempfile.TemporaryDirectory() as root:
-            model_cls, cfg, tokenizer, local_params_path, _ = get_backbone(name, root=root)
-            net = model_cls.from_cfg(cfg)
-            net.load_parameters(local_params_path)
-            net.hybridize()
-            num_params, num_fixed_params = count_parameters(net.collect_params())
-            assert num_params > 0
+@pytest.mark.parametrize('name', list_backbone_names())
```

Comment:
I moved to use sequential test intentionally because running multiple tests in parallel may cause some memory issues.

Reply:
In that case we may mark it as serial. Having a single large tests makes it very hard to reproduce failures of specific models, because the test will always test all models. It's not a good development experience.

Reply:
May be we should remove the forward test of xlmr, which is too large.